testing/camunda-process-test-java/src/test/java/io/camunda/process/test/impl/extensions/CamundaProcessTestContextIT.java

```diff
@@ -124,6 +129,10 @@ void shouldMockJobWorkerWithVariables() {
     assertThatProcessInstance(processInstanceEvent).isCompleted();
     assertThatProcessInstance(processInstanceEvent).hasCompletedElements("success-end");
     assertThatProcessInstance(processInstanceEvent).hasVariables(variables);
+
+    assertThat(mockedJobWorker.getInvocations()).isEqualTo(1);
+    assertThat(mockedJobWorker.getActivatedJobs().get(0).getVariablesAsMap())
+        .contains(entry("error_code", "404"));
```

Comment:
:wrench: We should add a few more test cases in `CamundaProcessTestContextIT`:
- No innovations
- Get innovations for `mockJobWorker("test").thenComplete()`
- Get innovations for `mockJobWorker("test").thenThrowBpmnError()`
- Get innovations for `mockJobWorker("test").withHandler()`